### PR TITLE
FLW-678 iOS. Staking. The back button hasn't offset.

### PR DESCRIPTION
**Pull request by stas-litvinov**
change back button appearance in Staking modules
(back button appearance in Wallet modules should be implemented as separate issue)

### CODE REVIEW

**Review by ERussel — CHANGES_REQUESTED** [review]
I think we need to update the Capital at the same time. I think we can add optional back button icon to the WalletNavigationBarStyleProtocol.

**Review by ef1rspb — APPROVED** [review]

File: fearless/Common/ViewController/NavigationController/FearlessNavigationController.swift [hunk 1]
```diff
@@ -32,6 +32,10 @@ final class FearlessNavigationController: UINavigationController, UINavigationCo
 
         navigationBar.setBackgroundImage(UIImage(), for: UIBarMetrics.default)
         navigationBar.shadowImage = UIImage()
+        navigationBar.backIndicatorImage = R.image.iconBack()
+        navigationBar.backIndicatorTransitionMaskImage = R.image.iconBack()
+
+        navigationItem.leftItemsSupplementBackButton = true
```
Comment (ERussel) [inline fearless/Common/ViewController/NavigationController/FearlessNavigationController.swift:38]: Navigation item is bonded to concrete controller. Does this work when we push new controller to the stack?

Comment (stas-litvinov) [reply fearless/Common/ViewController/NavigationController/FearlessNavigationController.swift:38]: Yes, it works like a charm
image
image

Comment (ERussel) [reply fearless/Common/ViewController/NavigationController/FearlessNavigationController.swift:38]: I tried to run and don't see any difference with this line of code ```navigationItem.leftItemsSupplementBackButton = true``` and without it. As expected navigationItem of the UINavigationController shouldn't make any effect to the navigation's bar items. Can you any difference? May be I am missing something

**Review by ERussel — APPROVED** [review]

**Pull request merged**